lib/resources/TaxIds.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+
+/**
+ * TaxIds is a unique resource in that, upon instantiation,
+ * requires a customerId, and therefore each of its methods only
+ * require the taxId argument.
+ *
+ * This streamlines the API specifically for the case of accessing a tax id
+ * on a returned customer object.
+ *
+ * E.g. customerObject.tax_id.retrieve(taxIdId)
```

Comment:
should this be `customerObject.tax_ids` (the id[s])?

Reply:
I'll be honest, this comment makes little sense to me (the library really needs to be cleaned-up/rewritten :p) but likely